tools/packaging/release/update-repository-version.sh

```diff
@@ -134,20 +130,14 @@ bump_repo() {
 	git fetch origin "${target_branch}"
 	git checkout "origin/${target_branch}" -b "${branch}"
 
-	# All repos we build should have a VERSION file
-	if [ ! -f "VERSION" ]; then
-		current_version="new"
-		echo "${new_version}" >VERSION
-	else
-		current_version="$(grep -v '#' ./VERSION)"
-
-		info "Updating VERSION file"
-		echo "${new_version}" >VERSION
-		if git diff --exit-code; then
-			info "${repo} already in version ${new_version}"
-			cat VERSION
-			return 0
-		fi
+	local current_version="$(egrep -v '^(#|$)' ./VERSION)"
+
+	info "Updating VERSION file"
+	echo "${new_version}" >VERSION
+	if git diff --exit-code; then
+		info "${repo} already in version ${new_version}"
+		cat VERSION
```

Comment:
Maybe you don't need this `cat` as the version is printed on the `info`.

Reply:
Removed!